exporter/gather.go

```diff
@@ -228,15 +262,15 @@ func logicalSwitchStateHander(resp *Nsxv3Response, data *Nsxv3Data) string {
 
 		data.LogicalSwitchesStates = append(data.LogicalSwitchesStates, *lswitchData)
 	}
-	return next
+	return next, nil
 }
 
 func init() {
-	endpoints = map[string]func(resp *Nsxv3Response, data *Nsxv3Data) string{
+	endpoints = map[string]func(resp *Nsxv3Response, data *Nsxv3Data) (string, error){
 		"/api/v1/cluster/status":         clusterStatusHandler,
-		"/api/v1/cluster/nodes/status":   clusterNodesStatusHandler,
-		"/api/v1/logical-switches":       logicalSwitchAdminStateHander,
-		"/api/v1/logical-switches/state": logicalSwitchStateHander,
+		"/api/v1/cluster/nodes/statufs":   clusterNodesStatusHandler,
+		"/api/v1/logical-switchefs":       logicalSwitchAdminStateHander,
+		"/api/v1/logical-switches/sftate": logicalSwitchStateHander,
```

Comment:
Is this intended?

Reply:
No, thanks for catching it, these were some of the test endpoints.